Assets/Scripts/Models/ContextMenu/ContextMenuAction.cs

```diff
@@ -2,7 +2,7 @@
 
 public class ContextMenuAction
 {
-    public bool RequiereCharacterSelected;
+    public bool RequireCharacterSelected;
```

Comment:
The public field was there long before but i guess i can change it to a property. Sorry for all of the stylecop errors but im pretty sure if i ran it on the files then i would be in for another few hours just fixing the whole file.